pytorch_lightning/plugins/environments/lsf_environment.py

```diff
@@ -51,112 +54,152 @@ def __init__(self):
+            jobid = os.environ.get(var)
+            if not jobid:
+                raise ValueError("Could not find job id -- expected in environment variable %s" % var)
+            else:
+                port = int(jobid)
+                # all ports should be in the 10k+ range
+                port = int(port) % 1000 + 10000
+            log.debug("calculated master port")
+        else:
+            log.debug("using externally specified master port")
+        return port
 
-    def set_world_size(self, size: int) -> None:
-        log.debug("LSFEnvironment.set_world_size was called, but setting world size is not allowed. Ignored.")
+    def _get_global_rank(self):
+        """A helper function for getting the global rank.
 
-    def global_rank(self):
-        """The world size is read from the environment variable `JSM_NAMESPACE_RANK`."""
+        Read this from the environment variable JSM_NAMESPACE_LOCAL_RANK
+        """
         var = "JSM_NAMESPACE_RANK"
         global_rank = os.environ.get(var)
         if global_rank is None:
             raise ValueError(
-                f"Cannot determine global rank from environment variable {var}."
-                " Make sure you run your executable with `jsrun`"
+                "Cannot determine global rank -- expected in %s "
+                "-- make sure you run your executable with jsrun" % var
             )
         return int(global_rank)
 
-    def set_global_rank(self, rank: int) -> None:
-        log.debug("LSFEnvironment.set_global_rank was called, but setting global rank is not allowed. Ignored.")
+    def _get_local_rank(self):
+        """A helper function for getting the local rank.
 
-    def local_rank(self):
-        """The local rank is read from the environment variable `JSM_NAMESPACE_LOCAL_RANK`."""
+        Read this from the environment variable JSM_NAMESPACE_LOCAL_RANK
+        """
         var = "JSM_NAMESPACE_LOCAL_RANK"
         local_rank = os.environ.get(var)
         if local_rank is None:
             raise ValueError(
-                f"Cannot determine local rank from environment variable {var}."
-                " Make sure you run your executable with `jsrun`"
+                "Cannot determine local rank -- expected in %s " "-- make sure you run your executable with jsrun" % var
             )
         return int(local_rank)
 
-    def node_rank(self):
-        """The node rank is determined by the position of the current hostname in the list of hosts stored in the
-        environment variable `LSB_HOSTS`."""
+    def _get_world_size(self):
+        """A helper function for getting the world size.
+
+        Read this from the environment variable JSM_NAMESPACE_SIZE
+        """
+        var = "JSM_NAMESPACE_SIZE"
+        world_size = os.environ.get(var)
+        if world_size is None:
+            raise ValueError(
+                "Cannot determine local rank -- expected in %s " "-- make sure you run your executable with jsrun" % var
```

Comment:
Let's use f"{var}" to be consistent with the rest of the codebase.

Reply:
```suggestion
                f'Cannot determine local rank -- expected in {var} -- make sure you run your executable with jsrun'
```